tests/ConfigTest.php

```diff
@@ -133,7 +133,7 @@ public function testThatCustomSymfonyFinderWorks()
         $config = Config::create()->setFinder($finder);
 
         $iterator = $config->getFinder()->getIterator();
-        $this->assertSame(1, count($iterator));
+        $this->assertCount(1, $iterator);
```

Comment:
@SpacePossum , fix to make it working properly on hhvm is to run `iterator_to_array($iterator)` first :/
still, even hhvm 3.18 is not fixed that bug... what a shame for hhvm...